docs/src/languages/python.md

```diff
@@ -15,142 +16,134 @@ description: Platform.sh supports deploying Python applications. Your applicatio
-   and a `setup.py` at [your app root](../create-apps/app-reference.md#root-directory),
-   create a hook like the following:
+2. Install the requirements for your app.
+   For example, to use `pipenv` to manage requirements and a virtual environment, add the following:
+
+   ```yaml {location=".platform.app.yaml"}
+   dependencies:
+       python3:
+           pipenv: "2022.5.2"
 
-   ```yaml
    hooks:
        build: |
-           pip install -r requirements.txt
-           pip install -e .
-           pip install gunicorn
+           pipenv install --system --deploy
    ```
 
-   These are installed as global dependencies in your environment.
+3. Define the command to start your web server:
 
-3. Configure the command you use to start serving your application (this must be a foreground-running process) under the `web` section, e.g.:
-
-   ```yaml
+   ```yaml {location=".platform.app.yaml"}
    web:
+       # Start your app with the configuration you define
+       # You can replace the file location with your location
        commands:
-           start: "gunicorn -b 0.0.0.0:$PORT project.wsgi:application"
+           start: python server.py
    ```
 
-   This assumes the WSGI file is `project/wsgi.py` and the WSGI application object is named `application` in the WSGI file.
+### Use uWSGI
+
+You can also use [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) to manage your server.
+Follow these steps to get your server started.
+
+1. Specify one of the [supported versions](#supported-versions):
+
+    {{< readFile file="/registry/images/examples/full/python.app.yaml" highlight="yaml" location=".platform.app.yaml" >}}
 
-4. Define the web locations your application is using:
+2. Define the conditions for your web server:
 
-   ```yaml
+   ```yaml {location=".platform.app.yaml"}
    web:
+       upstream:
+           # Send requests to the app server through a unix socket
+           # Its location is defined in the SOCKET environment variable
+           socket_family: "unix"
+
+       # Start your app with the configuration you define
+       # You can replace the file location with your location
+       commands:
+           start: "uwsgi --ini conf/uwsgi.ini"
+
        locations:
+           # The folder from which to serve static assets
            "/":
-               root: ""
+               root: "public"
                passthru: true
-               allow: false
-           "/static":
-               root: "static/"
-               allow: true
+               expires: 1h
    ```
 
-   This configuration asks our web server to handle HTTP requests at "/static" to serve static files stored in `/app/static/` folder while everything else is forwarded to your application server.
+3. Create configuration for uWSGI such as the following:
 
-5. Create any Read/Write mounts. The root file system is read only.  You must explicitly describe writable mounts.
+   ```ini {location="config/uwsgi.ini}
```

Comment:
:lady_beetle: The [file location thingy](https://pr-2393-pijhfna-652soceglkw4u.eu-3.platformsh.site/languages/python.html#use-uwsgi) doesn't work for ini files.

Reply:
It works, just missing the closing quotes